Task8/src/components/Search/Search.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
+
+import * as TodoActionCreators from '../../api/video';
+
+import './Search.css';
+
+class Search extends Component {
+  constructor(props) {
+    super(props);
+    this.props = props;
+    this.textInput = React.createRef();
+    console.log(this.props.theme);
+  }  
+
+  render() {
+    const { buttonName } = this.props;
+    return (
+      <div className="header__search">
+        <input
+          className="search__input"
+          type="text"
+          size="30"
+          ref={this.textInput}
+        />
+        <button
+          onClick={() => this.props.loadVideo()(this.textInput.current.value, false)}
```

Comment:
better to introduce handleSearch instead
from docs: `The problem with this syntax is that a different callback is created each time the Xxx renders`